care/emr/resources/scheduling/schedule/spec.py

```diff
@@ -77,8 +78,15 @@ def validate_for_slot_type(self):
                 end_time = datetime.datetime.combine(
                     datetime.datetime.now(tz=UTC).date(), availability.end_time
                 )
+                duration = (end_time - start_time).total_seconds()
```

Comment:
keep variables named verbose. duration is ambigious. slot's duration? availabilities duration? schedules duration?

```suggestion
                availability_duration_in_seconds = (end_time - start_time).total_seconds()
```